File: Entities/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace school_control_net.Entities
{
    public class Student : BaseEntity<int>
    {
        public string Name { get; set; }
        public string Address { get; set; }
        public DateTime DateOfBirth { get; set; }
        public IList<Course> Courses { get; set; }
    }
}

File: school-control-front/src/app/models/student.ts
export class Student
{
	id: number;
	name: string| null;
	address: string | null;
	dateOfBirth: Date | null;
	
	constructor() {
		this.id = 0;
		this.name = null;
		this.address = null;
		this.dateOfBirth = null;
	}
}